mlflow/models/python_api.py

```diff
@@ -94,6 +97,7 @@ def build_docker(
 _CONTENT_TYPE_JSON = "json"
 
 
+@experimental
```

Comment:
Q: Do we have a particular reason to mark this API experimental?

Reply:
As I'm changing the input_data to be 100% PyFunc.predict input data here, and not many people are using this API so just mark this `experimental` in case ppl feels like the behavior is changing :D 
Even though in the docstring we mentioned `input_data: Input data for prediction. Must be valid input for the PyFunc model.` but in reality for the old implementation, if it's a string, we expect input_data to be `{"inputs": "some_string"}` instead of `"some_string"`